test/com/twu/biblioteca/UserSessionTest.java

```diff
@@ -0,0 +1,15 @@
+package com.twu.biblioteca;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class UserSessionTest {
+
+    @Test
+    public void shouldValidateUser() {
+        UserSession userSession = new UserSession();
+        boolean validUser = userSession.validateUser("123-1234", "password");
+        System.out.print(validUser);
+        Assert.assertTrue(validUser);
+    }
```

Comment:
This is a good test (although the `System.out` is probably no longer needed), but whenever you have a boolean method you are testing, you should be thinking of _at least_ one test for each possible return value.